### DISCUSSION TITLE
Windows-2022 Image: 120.0.0 Edge Driver issues

### DISCUSSION THREAD

**Discussion by Ztomporo** [actions/virtual-environments#9130] [General questions]
I have a feeling I need to bring this up in the Edge driver repo. But it looks like there's a problem with the current latest windows-2022 image and edge driver version and edge version. Our tests are failing because they are looking for a new window to open up to (Sign In) that is no longer working with the latest edge driver. Works with version 119 and latest version of edge browser I have tested that.

Is there any way to go back in versions of the images like setting in our pipeline   windows-2022-1321.4 or something. while we wait for an updated image with an updated edge driver etc. 

If not all good!

Thanks!

**Comment by Alexey-Ayupov** [comment]
Hello @Ztomporo, unfortunately we have no such possibility. We are going to deploy new Windows-2022 image this week. Stay tuned.
